source/clear-linux/reference/collaboration/writing-guide.rst

```diff
@@ -0,0 +1,419 @@
+.. _writing-guide:
+
+Writing guide
+#############
+
+We want our documentation to be easy to read and understand. This document
+describes guidelines for writing documentation that is clear, concise,
+confident, and courteous.
+
+Refer to :ref:`structure-formatting` for details on organizing content and how
+we use reStructuredText and Sphinx.
+
+.. contents:: :local:
+   :depth: 1
+
+Use simple English
+******************
+
+Write using simple English: Be brief and communicate only the information that
+is needed. Be friendly and informative. Emphasize clarity and avoid
+unecessary complicated or technical terms. Make the content accessible to
+non-native speakers.
+
+Be brief
+========
+
+Use short sentences and paragraphs. Stick to the principle of one main
+idea per sentence, plus one additional point if needed. Each paragraph
+should address one main idea. Remember the basic structure of a paragraph:
+Introduction, body, and conclusion.
+
+Be friendly
+===========
+
+We write for our peers and want to be familiar. Use the second person, you or
+we. Be professional, respectful, and cooperative.

```

Comment:
"we" is 1st person plural. Maybe just "Use you or we."

Reply:
Done.